cmd/collectors/zapi/plugins/shelf/shelf.go

```diff
@@ -201,7 +209,7 @@ func (my *Shelf) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 						instance, err := data1.NewInstance(instanceKey)
 
 						if err != nil {
-							my.Logger.Debug().Msgf("add (%s) instance: %v", attribute, err)
+							my.Logger.Error().Msgf("add (%s) instance: %v", attribute, err)
```

Comment:
my.Logger.Error().Err(err).Str("attribute", attribute).Msg("Failed to add instance")